SPECS-EXTENDED/suitesparse/suitesparse.spec

```diff
@@ -0,0 +1,728 @@
+Summary:        A collection of sparse matrix libraries
+Name:           suitesparse
+Version:        5.4.0
+Release:        5%{?dist}
+License:        (LGPLv2+ OR BSD) AND LGPLv2+ AND GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://faculty.cse.tamu.edu/davis/suitesparse.html
+Source0:        https://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-%{version}.tar.gz
+%global amd_version_major 2
+%global btf_version_major 1
+%global camd_version_major 2
+%global ccolamd_version_major 2
+%global cholmod_version_major 3
+%global colamd_version_major 2
+%global csparse_version_major 3
+%global cxsparse_version_major 3
+%global klu_version_major 1
+%global ldl_version_major 2
+%global rbio_version_major 2
+%global spqr_version_major 2
+%global SuiteSparse_config_major 5
+%global umfpack_version_major 5
+### CXSparse is a superset of CSparse, and the two share common header
+### names, so it does not make sense to build both. CXSparse is built
+### by default, but CSparse can be built instead by defining
+### enable_csparse as 1 below.
+%global enable_csparse 0
+# Whether to build a separate version of libraries linked against an ILP64 BLAS
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  hardlink
+BuildRequires:  metis-devel
+BuildRequires:  openblas-devel
+BuildRequires:  tbb-devel
+Provides:       ufsparse = %{version}-%{release}
+
+%description
+suitesparse is a collection of libraries for computations involving sparse
+matrices.  The package includes the following libraries:
+  AMD                 approximate minimum degree ordering
+  BTF                 permutation to block triangular form (beta)
+  CAMD                constrained approximate minimum degree ordering
+  COLAMD              column approximate minimum degree ordering
+  CCOLAMD             constrained column approximate minimum degree ordering
+  CHOLMOD             sparse Cholesky factorization
+  CSparse             a concise sparse matrix package
+  CXSparse            CSparse extended: complex matrix, int and long int support
+  KLU                 sparse LU factorization, primarily for circuit simulation
+  LDL                 a simple LDL factorization
+  SQPR                a multithread, multifrontal, rank-revealing sparse QR
+                      factorization method
+  UMFPACK             sparse LU factorization
+  SuiteSparse_config  configuration file for all the above packages.
+  RBio                read/write files in Rutherford/Boeing format
+
+%package devel
+Summary:        Development headers for SuiteSparse
+Requires:       %{name} = %{version}-%{release}
+Provides:       ufsparse-devel = %{version}-%{release}
+
+%description devel
+The suitesparse-devel package contains files needed for developing
+applications which use the suitesparse libraries.
+
+%package static
```

Comment:
Can we exclude the static library?  Prefer to keep those out of Mariner as much as possible

Reply:
Given it is a HPC based library some applications may need the static libraries. I think should ship them :)